tests/rkt_tests.go

```diff
@@ -249,6 +249,31 @@ func patchTestACI(newFileName string, args ...string) string {
 	return patchACI(image, newFileName, args...)
 }
 
+func spawnOrFail(t *testing.T, cmd string) *gexpect.ExpectSubprocess {
+	child, err := gexpect.Spawn(cmd)
+	if err != nil {
+		t.Fatalf("Cannot exec rkt: %v", err)
+	}
+	return child
+}
+
+func waitOrFail(t *testing.T, child *gexpect.ExpectSubprocess, shouldSucceed bool) {
```

Comment:
Bikeshedding here:

I don't like this boolean parameter - I'd prefer some named values instead like:
```go
type expectedOutcome int // or errorStatus or whatever
const (
	shouldFail expectedOutcome = iota
	shouldFail
)

func waitOrFail(t *testing.T, child *gexpect.ExpectSubprocess, outcome expectedOutcome) {...}

// calling
waitOrFail(t, child, shouldSucceed)
```